ocean_lib/web3_internal/wallet.py

```diff
@@ -123,20 +123,26 @@ def sign_tx(
             )
         else:
             nonce = Wallet._get_nonce(self.web3, account.address)
+            from ocean_lib.web3_internal.contract_base import ContractBase
```

Comment:
you can move the needed function somewhere in utils to avoid the import error

Reply:
I have tried right now to move into web3_internal/utils and also in web3_internal/transactions (because it fits better in this folder since it's related to web3) and it's still complaining about the conftest (circular import errors).

Reply:
I'm sure we can find a way. It is a basic function not using any higher level components, so we should be able to move it somewhere safe.